.snippets/text/node-operators/block-producers/run-a-block-producer/docker-command.md

```diff
@@ -11,7 +11,7 @@
 --base-path=/data/container \
 -- \
 --name=INSERT_YOUR_RELAY_NODE_NAME \
---chain=/chain-network/westend-raw-specs.json \
+--chain=westend_moonbase_relay_testnet \
```

Comment:
these are the changes coming in runtime 500?

Reply:
Indeed. (At the moment the specs are not included in the image, and the only way to make that work would be to download them manually)